### DISCUSSION TITLE
Admin Panel making incorrect GET requests after adding SSL certificate

### DISCUSSION THREAD

**Discussion by andrewsrichardson** [strapi/strapi#8012] [Legacy Topics]
Hello, I have followed instructions on getting a strapi server SSL certified using LetsEncrypt for a digital ocean droplet using nginx. The redirect works fine and the content is now served over HTTPS via the strapi API. However, when accessing the admin panel at my new domain, I am met with an infinite load wheel as the request is made to the IP address of the droplet:

![image](https://user-images.githubusercontent.com/52127531/94036784-985afc00-fdbc-11ea-8025-93e41c5e08e6.png)

Also, when I hit the base url, then click through to access the admin panel, I am met with a 404 error as the resource is being requested from the IP, not the new domain.

And when I restart the strapi server, I can see that it is running at the IP of my droplet:

![image](https://user-images.githubusercontent.com/52127531/94038468-6185e580-fdbe-11ea-8028-52d0d5540c2e.png)

So I think there must be a strapi setting which I need to change to tell the strapi server that I have changed the root domain for subsequent calls - but I have not seen this documented anywhere. Any help is appreciated! I am extremely new to server technologies so it is likely that I have simply missed something

**Comment by derrickmehaffy** [comment]
I think I probably need to add a bit of docs in our DO one-click guide for this (adding it to my to-do) but in `/srv/strapi/strapi-development` there is a `.env` file with an NGINX_URL environment variable, you will need to change this over to whatever your public domain is (including the `https://`) then trigger a rebuild of the admin panel for it to take effect

**Reply by andrewsrichardson** [reply]
How do I go about triggering a rebuild? I cant find anywhere in the DO console which allows me to run Strapi commands (specifically Strapi build)

**Reply by derrickmehaffy** [reply]
change over to the directory for the Strapi application `/srv/strapi/strapi-development` and run either `npm run build` or `yarn build` (yarn is installed by default, and it's what we recommend)

**Reply by derrickmehaffy** [reply]
You will have to use SSH and change over to the service user: https://strapi.io/documentation/v3.x/installation/digitalocean-one-click.html#using-the-service-account

**Reply by andrewsrichardson** [reply]
Perfect, thanks alot for your help
